records_mover/db/postgres/copy_options/date_output_style.py

```diff
@@ -0,0 +1,40 @@
+from records_mover.utils import quiet_remove
+from records_mover.records.types import RecordsHints
+from records_mover.records.hints import cant_handle_hint
+from typing import Set, Tuple, Optional
+from .types import DateOrderStyle, DateOutputStyle
+
+
+def determine_date_output_style(unhandled_hints: Set[str],
+                                hints: RecordsHints,
+                                fail_if_cant_handle_hint: bool) -> \
+        Tuple[DateOutputStyle, Optional[DateOrderStyle]]:
```

Comment:
The semantics of the date-related commands is fundamentally different, though - Postgres is very limited on date output, but very forgiving on date input, so I broke the handling of them out into two separate input/output functions.